### PR TITLE
content/news: Add post for April Fools Day 2024

### PR DESCRIPTION
**Pull request by Holzhaus**
stonks

### CODE REVIEW

File: content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+comments: yes
+
+
+The Mixxx Board of Directors in pleased to announce plans to acquire the AlphaTheta Corporation (formerly known as Pioneer DJ), Serato Audio Research Ltd. and Native Instruments.
+
+*"This exciting consolidation of brands will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.
+Some trademarks such as Rekordbox®, Serato® and Traktor® will likely be dropped to archieve a lean portfolio and focus on the popular Mixxx brand.
+
+The move will lead to new synergies and enables Mixxx to benefit from the know-how of its former competitors.
+*"Our new subsidiaries have a track record in profitable partnerships with hardware manufacturers."*, McStrach commented.
```
Comment (JoergAtGithub) [inline content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md]:
```suggestion
*"Our new subsidiaries have a track record in profitable partnerships with hardware manufacturers."*, McScratch commented.
```

File: content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+The Mixxx Board of Directors in pleased to announce plans to acquire the AlphaTheta Corporation (formerly known as Pioneer DJ), Serato Audio Research Ltd. and Native Instruments.
+
+*"This exciting consolidation of brands will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.
+Some trademarks such as Rekordbox®, Serato® and Traktor® will likely be dropped to archieve a lean portfolio and focus on the popular Mixxx brand.
+
+The move will lead to new synergies and enables Mixxx to benefit from the know-how of its former competitors.
+*"Our new subsidiaries have a track record in profitable partnerships with hardware manufacturers."*, McScratch commented.
+*"We want to leverage their expertise to lock down DJ hardware and ensure that the majority of DJ equipment in the market will only work with Mixxx. We are committed to our customers, so it's only fair to demand that they stay committed to us as well."*
```
Comment (Holzhaus) [inline content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md]:
```suggestion
*"We want to leverage their expertise on locking down DJ hardware artificially and ensure that the majority of DJ equipment in the market will only work with Mixxx. We are committed to our customers, so it's only fair to demand that they stay committed to us as well."*
```

**Review by Swiftb0y — COMMENTED** [review]
I'd end it with a little "happy april 1st", just in case. expressing sarcasm purely via text doesn't always work, so I'd prefer to spell it out for the people that don't get it.

**Comment by Holzhaus** [comment]
> I'd end it with a little "happy april 1st", just in case. expressing sarcasm purely via text doesn't always work, so I'd prefer to spell it out for the people that don't get it. 

I mean part of the fun is also that we're are always few people who don't get it at first. And here it's already pretty obvious that this is a joke, and people will notice that at least when reading the comments I guess.

If you want we can add "april fools" to the tags, that's a bit less obvious than stating it in the text explicitly.

**Comment by daschuer** [comment]
I would prefer to have no explicit hint on the 1st of April. But we need to consider how to continue in the other days. Just delete it or add a disclaimer?

**Comment by Swiftb0y** [comment]
I guess lets stick to what we did [in 2021](https://mixxx.org/news/2021-04-01-mixxx-subscription-model/): add an april fools tag and add an edit later?

**Comment by daschuer** [comment]
We have added the tag including the edit later:
https://github.com/mixxxdj/website/pull/228/files
Let's stick to it.

**Comment by Swiftb0y** [comment]
sure then... Well, whats the state then? This still feels kinda short. @Holzhaus do you (or any1 else) have interest/time to write a little more?

**Comment by Holzhaus** [comment]
> This still feels kinda short. @Holzhaus do you (or any1 else) have interest/time to write a little more?

I agree this is a bit short, but I'm out of ideas. If you have something that you'd like to get in, please make a suggestion.

**Comment by ronso0** [comment]
I'm also not in that creative mood to add something worthy.

It's short and that's okay I think.

@Holzhaus Thanks for writing this!

**Comment by Swiftb0y** [comment]
this is supposed to critique the AlphaTheta takeover, maybe we can add another paragraph somewhere that mentions the issue specifically by linking to an article highlighting the issue. mby https://www.thepost.co.nz/a/business/350059653/us-firm-waves-prospect-legal-action-stop-100m-buy-out-nz-software-firm-serato

**Review by ywwg — APPROVED** [review]

File: content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+title: Mixxx Plans to Acquire AlphaTheta, Serato and Native Instruments
+authors: Jan Holthuis
+tags: takeover, super important business decision
+status: draft
+comments: yes
+
+
+The Mixxx Board of Directors in pleased to announce plans to acquire the AlphaTheta Corporation (formerly known as Pioneer DJ), Serato Audio Research Ltd. and Native Instruments.
+
+*"This exciting consolidation of brands will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.
```
Comment (Holzhaus) [inline content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md]:
```suggestion
*"This exciting [consolidation of brands](https://www.thepost.co.nz/a/business/350059653/us-firm-waves-prospect-legal-action-stop-100m-buy-out-nz-software-firm-serato) will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.
```
@Swiftb0y like this?

**Comment by Holzhaus** [comment]
Fro my side this is ready to merge.

**Review by Swiftb0y — APPROVED** [review]
LGTM. waiting for build. Merging so we still hit the april 1st deadline.

**Pull request merged**